### PR TITLE
Updates to homepage and TribTalk style guide

### PR DESCRIPTION
**Pull request by kabeaty**
#### What's the plan?

To update some elements on homepage and TribTalk style guide page for enhanced look and more info.
#### What's this PR do?

Adds in goals/statements copy to homepage for more welcoming info. Adds example code toggles to TribTalk page. Some various styling and spacing adjustments made to homepage and TribTalk style guide as well; see issues linked below under relevant tickets for a more detailed breakdown.
#### How should this be manually tested?

Run styleguides locally and click around to see that everything is looking good.
#### What are the relevant tickets?

This Sprint task: https://basecamp.com/1866004/projects/1175549/todos/119956247
These GitHub issue #10 and #11.
#### Smells?

Don't think so...
#### TODOs:
- [x] Update so that welcome copy isn't included on every page
- [x] Add image to homepage
- [x] Add copy to TribTalk branding section
- [x] Improve TribTalk typography section by fleshing it out a bit more
- [x] Smooth out spacing between sections/headers on TribTalk
#### Technical debt note

Adds more code.

### CODE REVIEW

**Comment by crccheck** [comment]
can you install the editorconfig plugin for your editor? http://editorconfig.org/#download

**Comment by kabeaty** [comment]
Yes, installed the editorconfig plugin and had the .editorconfig file.

File: _includes/foot.html [hunk 1]
```diff
@@ -1,2 +1,14 @@
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
+    <script>
+
+        //Toggles to show/hide example code on button click
+        $(document).ready(function() {
```
Comment (crccheck) [inline _includes/foot.html]: was wondering if you had .editorconfig because these new lines are 4-space indented instead of 2-space

**Comment by kabeaty** [comment]
OK, thanks. I fixed the spacing on foot.html, and now that I have the plugin installed hopefully everything should be good to go in future.

**Comment by risatrix** [comment]
Looks good to me.

**Pull request merged**